### DISCUSSION TITLE
Imágenes para la practica

### DISCUSSION THREAD

**Discussion by zarry0** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#374] [Ideas]
Buenas tardes compañeros, les anexo un link con la imágenes que el diseñador original utilizó. Espero les sirva, me parece que son de uso libre siempre y cuando se otorguen los créditos apropiados al creador original.

https://www.freepik.com/free-photo/front-view-female-nurse-white-medical-suit-with-red-mask-injection-her-hands-blue_13935723.htm?query=nurses%20%20nursing#position=30&query=nurses++nursing

**Comment by IxshelC** [comment]
![enfermera2](https://user-images.githubusercontent.com/87400845/158043400-4e61ba68-49b1-4967-901f-80436161e101.png)
![Enfermero](https://user-images.githubusercontent.com/87400845/158043402-6563cee1-7ade-4957-9789-41fb7288fcda.png)

Les adjunto las otras dos :D por si las ocupan

**Reply by JossPerezT** [reply]
Muchas gracias! 
Me has ahorrado el acomodarlas con el fondo! Aún no lo lograba 😅🚀

**Comment by AlexAntonioG** [comment]
Muchas gracias, las necesitaba 🥇😄

**Comment by JossPerezT** [comment]
Hola. 
¿Sabes cómo se le deben dar los créditos correspondientes? 
Solo se le pone una leyenda al final de página, o cómo?

**Comment by Gracesl** [comment]
mil gracias me faltaban las imagenes
